internal/git/git.go

```diff
@@ -16,26 +18,64 @@ const (
 
 // copied from goreleaser
 
-// IsRepo returns true if current folder is a git repository
-func IsRepo() bool {
-	out, err := run("rev-parse", "--is-inside-work-tree")
-	return err == nil && strings.TrimSpace(out) == "true"
+type Repository struct {
+	GitWorkTree  string
+	GitDirectory string
 }
```

Comment:
I'm sorry but I didn't notice all this was in the git package

Maybe using git in git.Repository wasn't the best idea. Maybe @caarlos0 will provide some feedbacks here